specification/communication/data-plane/CallAutomation/preview/2023-01-15-preview/communicationservicescallautomation.json

```diff
@@ -2615,56 +2603,6 @@
         }
       }
     },
-    "SendDtmfCompleted": {
-      "type": "object",
-      "properties": {
-        "callConnectionId": {
-          "description": "Call connection ID.",
-          "type": "string"
-        },
-        "serverCallId": {
-          "description": "Server call ID.",
-          "type": "string"
-        },
-        "correlationId": {
-          "description": "Correlation ID for event to call correlation. Also called ChainId for skype chain ID.",
-          "type": "string"
-        },
-        "operationContext": {
-          "description": "Used by customers when calling mid-call actions to correlate the request to the response event.",
-          "type": "string"
-        },
-        "resultInformation": {
-          "$ref": "#/definitions/ResultInformation",
-          "description": "Contains the resulting SIP code/sub-code and message from NGC service."
```

Comment:
discussed offline - at the moment, it's the same message for all events having resultInformation, including the GA contracts. It will be taken care separately. 
Resolving comment to merge PR.